src/MicrosoftTeams.ts

```diff
@@ -1987,7 +1987,17 @@ namespace microsoftTeams {
     /**
      * JSON defining an adaptive card.
      */
-    card?: string;
+    card?: any;
+
+    /**
```

Comment:
This is feature creep and not in scope of this feature. Please remove. This needs to go thorough a thorough design + security review before we even attempt to build it.